src/gui/properties/speedplotview.cpp

```diff
@@ -202,6 +204,65 @@ quint64 SpeedPlotView::maxYValue()
     return maxYValue;
 }
 
+namespace
```

Comment:
shouldn't we have namespace declaration at top of the file?

Reply:
moved